procwrap/procwrap.py

```diff
@@ -2,39 +2,157 @@
 
 import os
```

Comment:
@algrebe Ah. Yes. That :) I believe we can do it with deeper surgery. Let us discuss on call today.